src/core/services/compiler/compiler.js

```diff
@@ -124,7 +124,8 @@ function MdCompilerProvider($compileProvider) {
    * - `>=1.7` - the compiler calls the constructor first before assigning bindings and
    *    `$compileProvider.preAssignBindingsEnabled()` no longer exists.
    *
-   * The default value is `false` but will change to `true` in AngularJS Material 1.2.
+   * The default value is `false` in AngularJS 1.6 and earlier but `true` in AngularJS 1.7.
+   * It is planned to change this to always default to `true` in AngularJS Material 1.2.
```

Comment:
Saying that the default is `true` in AngularJS 1.7 suggests it can be changed, especially that the docs nowhere say that you can't change it.

Also, `AngularJS 1.7` -> `AngularJS 1.7 or later`.

How about something like:
```
In AngularJS 1.7 or later the value is hardcoded to `true`. In AngularJS 1.6 and earlier
it's `false` by default. It is planned to change this to always default to `true` in
AngularJS Material 1.2.
```